src/class/hid/hid_device.c

```diff
@@ -232,6 +232,7 @@ bool hidd_control_request(uint8_t rhport, tusb_control_request_t const * request
 
   #if CFG_TUD_HID>1
   uint8_t const hid_itf = get_hid_index_by_itfnum((uint8_t) request->wIndex);
+  TU_VERIFY(hid_itf<0xFF, 0);
```

Comment:
since return value is bool, you should return false instead of 0. or you could just simply drop it and just leave 

```
TU_VERIFY(hid_itf<0xFF);
```

TU_VERFIY has a trick to have 2nd arg as optional.